src/utilities/awala/InternetEndpoint.spec.ts

```diff
@@ -255,7 +255,7 @@ describe('InternetEndpoint instance', () => {
       senderCertificate = await issueEndpointCertificate({
         issuerPrivateKey: senderKeyPair.privateKey,
         subjectPublicKey: senderKeyPair.publicKey,
-        validityEndDate: addSeconds(new Date(), 1),
+        validityEndDate: addMinutes(new Date(), 1),
```

Comment:
This is too short, so it make tests flaky on CI (which is slower than our computers).